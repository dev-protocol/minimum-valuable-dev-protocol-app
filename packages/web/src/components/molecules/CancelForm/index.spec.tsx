import React from 'react'
import { render } from '@testing-library/react'
import { CancelForm } from '.'
import 'src/__mocks__/window/matchMedia.mock'

describe(`${CancelForm.name}`, () => {
  test('Snapshot', () => {
    const component = render(
      <CancelForm remainBlocks={1} isCompleted={false} onClickCancel={() => {}} onClickWithdraw={() => {}} />
    )
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
