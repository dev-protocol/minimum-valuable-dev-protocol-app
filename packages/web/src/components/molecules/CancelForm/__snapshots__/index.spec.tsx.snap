// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CancelForm Snapshot 1`] = `
<body>
  <div>
    <div
      class="dy1yq4-0"
    >
      <p>
        Cancel Staking
      </p>
      <button
        class="ant-btn ant-btn-primary ant-btn-lg"
        type="button"
      >
<<<<<<< HEAD
        <div
          class="ant-space-item"
        >
          <div
            style="display: flex;"
          >
            <div
              style="margin-right: 5px;"
            >
              <button
                class="ant-btn ant-btn-primary ant-btn-lg"
                type="button"
              >
                <span>
                  Cancel
                </span>
              </button>
            </div>
            <button
              class="ant-btn ant-btn-primary ant-btn-lg"
              disabled=""
              type="button"
            >
              <span>
                Withdraw
              </span>
            </button>
            <div>
              <div
                class="ant-spin ant-spin-spinning"
                style="margin-right: 8px;"
              >
                <span
                  aria-label="loading"
                  class="anticon anticon-loading ant-spin-dot"
                  role="img"
                  style="font-size: 18px;"
                >
                  <svg
                    aria-hidden="true"
                    class="anticon-spin"
                    data-icon="loading"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="0 0 1024 1024"
                    width="1em"
                  >
                    <path
                      d="M988 548c-19.9 0-36-16.1-36-36 0-59.4-11.6-117-34.6-171.3a440.45 440.45 0 00-94.3-139.9 437.71 437.71 0 00-139.9-94.3C629 83.6 571.4 72 512 72c-19.9 0-36-16.1-36-36s16.1-36 36-36c69.1 0 136.2 13.5 199.3 40.3C772.3 66 827 103 874 150c47 47 83.9 101.8 109.7 162.7 26.7 63.1 40.2 130.2 40.2 199.3.1 19.9-16 36-35.9 36z"
                    />
                  </svg>
                </span>
              </div>
              1
               block(s) left
            </div>
          </div>
        </div>
      </div>
=======
        <span>
          Cancel
        </span>
      </button>
>>>>>>> upstream/main
    </div>
  </div>
</body>
`;
